src/main/java/org/jenkinsci/plugins/vsphere/parameters/CloudSelectorParameter.java

```diff
@@ -1,21 +1,20 @@
 package org.jenkinsci.plugins.vsphere.parameters;
 
-import static org.jenkinsci.plugins.vsphere.tools.PermissionUtils.throwUnlessUserHasPermissionToAccessJob;
-
 import hudson.Extension;
 import hudson.model.Item;
 import hudson.model.ParameterValue;
 import hudson.model.SimpleParameterDefinition;
 import hudson.model.StringParameterValue;
 import hudson.util.ListBoxModel;
+import java.util.List;
 import net.sf.json.JSONObject;
 import org.jenkinsci.plugins.vSphereCloud;
 import org.kohsuke.stapler.AncestorInPath;
 import org.kohsuke.stapler.DataBoundConstructor;
 import org.kohsuke.stapler.StaplerRequest;
 import org.kohsuke.stapler.export.Exported;
 
-import java.util.List;
+import static org.jenkinsci.plugins.vsphere.tools.PermissionUtils.throwUnlessUserHasPermissionToAccessJob;
```

Comment:
Unnecessary code churn - this just makes it harder to merge the PR with other PRs.
In general, a PR should EITHER make functional changes (and keep code changes to the minimum required to do that) so that it's as easy as possible to do a manual merge with other changes OR make formatting changes (that don't affect functionality at all) so that folks can achieve the same effect on their own code knowing it changes nothing important - don't do both at once.
(personally, I prefer static imports at the top - I suspect your IDE is configured differently and did this on your behalf...)

Reply:
Yes, blame IDE 😅

Reply:
reverted